www/front_src/src/Resources/index.tsx

```diff
@@ -129,14 +137,30 @@ const Resources = (): JSX.Element => {
       .finally(() => setLoading(false));
   };
 
+  const initAutorefresh = (): void => {
+    window.clearInterval(refreshIntervalRef.current);
+
+    const interval = enabledAutorefresh
+      ? window.setInterval(load, refreshIntervalMs)
```

Comment:
```suggestion
      ? setInterval(load, refreshIntervalMs)
```

Reply:
typescript does not work with this change